_rules/aria-state-or-property-valid-value-6a7281.md

```diff
@@ -59,39 +59,39 @@ Some user agents treat the value of `aria-*` attribute as case-sensitive (even w
 
 #### Passed Example 1
 
-`aria-required` property with valid true/false value
+This `div` element has an `aria-required` property with valid `true/false` value.
 
 ```html
 <div role="textbox" aria-required="true" aria-label="A required textbox"></div>
 ```
 
 #### Passed Example 2
 
-`aria-expanded` state with valid true/false/undefined value
+This `div` element has an `aria-expanded` state with valid `true/false/undefined` value.
 
 ```html
 <div role="button" aria-expanded="undefined">A button</div>
 ```
 
 #### Passed Example 3
 
-`aria-pressed` state with valid tristate value
+This `div` element has an `aria-pressed` state with valid tristate value.
 
 ```html
 <div role="button" aria-pressed="mixed">An other button</div>
 ```
 
 #### Passed Example 4
 
-`aria-errormessage` property with valid ID reference value
+This `div` element has an `aria-errormessage` property with valid ID reference value.
 
 ```html
 <div role="textbox" aria-errormessage="my-error" aria-label="A textbox"></div>
 ```
 
 #### Passed Example 5
 
-`aria-owns` property with valid ID reference list value
+This `div` element has an `aria-owns` property with valid ID reference list value.
```

Comment:
There are 3 `div` in that example, so "**this** `div`" is not well defined.

Plus, add explanation on whether the IDs need to exist or not.